.github/workflows/stable-cicd.yaml

```diff
@@ -85,8 +85,8 @@ jobs:
                 name: 🫙 Jar and Tag Determination
                 id: jartag
                 run: |
-                    echo "::set-output name=jar_file::$(ls ./service/target/*.jar)"
-                    echo "::set-output name=image_tag::$(echo ${{github.ref}} | awk -F/ '{print $NF}')"
+                    echo "jar_file=$(ls ./service/target/*.jar)" >> $GITHUB_OUTPUT
```

Comment:
This should be `./service/target/registry-api-service-*.jar` because there are also `-bin.jar`, `-sources.jar`, `-docs.jar`, etc. You want it to be as specific as possible and match just one file—the one with the compiled code.

Reply:
fixed